lib/services/abilities/message_abilities.ts

```diff
@@ -104,41 +113,20 @@ export function messageAbilities(staff: Staff) {
 
     await conversations.joinConversation(conversationId)
 
-    if (conversation.type !== ConversationType.CUSTOMER || !config) {
-      return db.message.create({ data: unsaveMessage }); // No Sunshine
-    }
-
-    let sunshineMessageId = null
-
-    if (conversation.sunshineConversationId) {
-      const { messages } = await sunshineMessages.postMessage(
-        config.smoochAppId,
-        conversation.sunshineConversationId,
-        {
-          "author": {
-            "type": "business",
-            "displayName": config.appName
-          },
-          "content": content
-        }
-      );
-
-      sunshineMessageId = messages[0].id;
-    }
-
     //
     // Note:
     // We do an upsert to handle the scenario where a webhook is fired fast enough to
     // generate a race condition
     //
-    return await db.message.upsert({
-      where: { sunshineMessageId },
-      update: {},
-      create: {
-        ...unsaveMessage,
-        sunshineMessageId
-      }
+    const message = await db.message.create({
```

Comment:
yup, now we _can_ do that. Because the failure from `postMessage` would appear as a status on the message if it does fail.

(note we only send to sunshine for `type: CUSTOMER`